app/api/util.ts

```diff
@@ -96,6 +96,9 @@ const instanceActions: Record<string, InstanceState[]> = {
   // https://github.com/oxidecomputer/omicron/blob/6dd9802/nexus/db-queries/src/db/datastore/instance.rs#L865
   delete: ['stopped', 'failed'],
 
+  // TODO: whence my licence to make such a claim
+  update: ['stopped'],
```

Comment:
```suggestion
  // https://github.com/oxidecomputer/omicron/blob/144e91a/nexus/db-queries/src/db/datastore/instance.rs#L1039-L1043
  update: ['stopped', 'failed'],
```

failed's especially important so you can unset a boot disk that you want to detach failed instance, and i expect this to continue to be valid (even if not useful) when you can update other bits of an instance

Reply:
Thanks. Already had this done but not pushed